src/main/java/com/julienvey/trello/impl/TrelloImpl.java

```diff
@@ -479,41 +452,79 @@ public void addUrlAttachmentToCard(String idCard, String url) {
         postForObject(createUrl(ADD_ATTACHMENT_TO_CARD).asString(), new Attachment(url), Attachment.class, idCard);
     }
 
+    @Override
+    public List<Member> addMemberToCard(String idCard, String userId) {
+        List<Member> members = Arrays.asList(postForObject(createUrl(ADD_MEMBER_TO_CARD).asString(),
+                new StringValueHolder(userId), Member[].class, idCard));
+        return setInternalTrello(members);
+    }
+
+    @Override
+    public List<Member> removeMemberFromCard(String idCard, String userId) {
+        List<Member> members = Arrays.asList(delete(createUrl(REMOVE_MEMBER_FROM_CARD).asString(), Member[].class,
+                idCard, userId));
+        return setInternalTrello(members);
+    }
+
     /* internal methods */
 
     private <T> T postFileForObject(String url, File file, Class<T> objectClass, String... params) {
-        logger.debug("PostFileForObject request on Trello API at url {} for class {} with params {}", url,
-                objectClass.getCanonicalName(), params);
+        if (logger.isDebugEnabled()) {
+            logger.debug("PostFileForObject request on Trello API at url {} for class {} with params {}", url,
```

Comment:
let's skip debug checks before calling logger. the main value for that would be if a param was expensive to calculate, which is not the case for this library.